public/Import-DbaCsv.ps1

```diff
@@ -636,6 +679,41 @@ function Import-DbaCsv {
                             $BufferSize,
                             $NullValue
                         )
+                        if ($FirstRowHeader -and $shouldMapCorrectTypes) {
+                            # we can get default columns, all strings. This "fills" the $reader.Columns list, that we use later
+                            $null = $reader.GetFieldHeaders()
+                            # we get the table definition
+                            # we do not use $server because the connection is active here
+                            $tableDef = Get-DbaDbTable $instance -SqlCredential $SqlCredential -Database $Database -Table $table -Schema $schema
+
+                            if ($tableDef.Count -ne 1) {
+                                Stop-Function -Message "Could not fetch table definition for table $table in schema $schema" -ErrorRecord $_
```

Comment:
I'm unsure if this is the right thing to do here ... at this point I don't see ay branches in code that could allow reaching here and not having a table present, but ... "what if"?

Reply:
I'd leave it cuz we encounter "what if" more than we'd expect.